test/core/table.wast

```diff
@@ -6,7 +6,7 @@
 (module (table 0 1 funcref))
 (module (table 1 256 funcref))
 (module (table 0 65536 funcref))
-(module (table 0 0xffff_ffff funcref))
+(module (table 0 0x8f_ffff funcref))
```

Comment:
The limitation you cite concerns the concrete (and web-specific!) _dynamic_ limit, not the static one. So it merely means that you cannot successfully grow beyond that. It is still a valid module -- validity must not be environment-dependent.

Reply:
As far as I understand the spec text there, this is not a dynamic limit. The text says: 

> An implementation must reject a module that exceeds these limits with a {{CompileError}}.

Reply:
Hm, that's an oversight/bug in the JS API spec then. The core spec explicitly lists table size under [execution limitations](https://webassembly.github.io/spec/core/appendix/implementation.html#execution).

Reply:
I created a PR now that resolves that inconsistency, see https://github.com/WebAssembly/spec/pull/1195. Can we change the value here until the inconsistency in the spec is resolved?

Reply:
Thanks for PR! It seems like we should be able to land it today, so maybe we can defer this PR for a few hours and avoid a temporary hack? :)

Reply:
Ack. I reverted the change to this test.